extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthProcessor.java

```diff
@@ -176,6 +178,10 @@ public void defineHealthRoutes(BuildProducer<RouteBuildItem> routes,
             BeanArchiveIndexBuildItem beanArchiveIndex) {
         IndexView index = beanArchiveIndex.getIndex();
 
+        // log a warning if users try to use @Liveness or @Readiness with JAX-RS @Path
+        warnIfJaxRsPathUsed(index, LIVENESS);
+        warnIfJaxRsPathUsed(index, READINESS);
```

Comment:
MP Health 3.0 will remove the deprecated `@Health` annotation, but while we still have it I think we should log a warning for that one too?

Reply:
Sure thing

Reply:
Done